vhdbuilder/scripts/automate_release_notes.sh

```diff
@@ -33,6 +33,10 @@ generate_release_notes() {
         else
             go run vhdbuilder/release-notes/autonotes/main.go --build $build_id --date $image_version --include ${included_skus%?}
         fi
+        if [ $? -ne 0 ]; then
```

Comment:
with `set -e` configured, will this ever be reached ?

Reply:
good catch

Reply:
fixed